api/v1alpha/framework/imagerunner.schema.json

```diff
@@ -197,6 +197,32 @@
       "required": [
         "name"
       ]
+    },
+    "reporters": {
+      "description": "Supported saucectl reporters.",
+      "type": "object",
+      "properties": {
+        "json": {
+          "type": "object",
+          "description": "The JSON reporter merges test results from all jobs in the JSON format into a single report.",
```

Comment:
> merges test results from all jobs
Does it now? I don't think so. We don't know what's going on inside the container.

Reply:
`test results` mixed with `jobs` can be confusing in the context of a SO runner, because we don't know anything about the tests. It sounds like this json reporter will collect all test results (this is what it says) from all jobs (Sauce jobs) that the image runner kicks off. None of it is actually true.